api/urls.py

```diff
@@ -8,5 +8,5 @@
     path('api/group/<str:event>/<int:group>', views.get_group, name='getgroup'),
     path('api/group/join/<str:event>/<int:group>', views.join_group, name='joingroup'),
     path('api/group/leave/<str:event>/<int:group>', views.leave_group, name='leavegroup'),
-
+    path('metrics', views.metrics, name='metrics'),
```

Comment:
So this was because the "default" path for prometheus-format metrics is `/metrics` - so this means you can just stick the domain in the config and not configure a non-standard path. If you'd prefer it fit in with the app url scheme though, that's fine.